src/commcare_cloud/fab/fabfile.py

```diff
@@ -221,6 +221,7 @@ def env_common():
     all = servers['all']
     proxy = servers['proxy']
     webworkers = servers['webworkers']
+    manage = webworkers[0]
```

Comment:
thoughts on making this `servers.get('manage', webworkers[0])` in case we want it to be overridden? (for example on ICDS, webworkers[0] is also the proxy and already has some high memory, so may want to do it somehwere else